### ISSUE TITLE
Trying to use external HDD as internal.

### ISSUE THREAD

**Issue by AkramHamd** [fbelavenuto/arpl-addons#96]
So I have this machine:

![imagen](https://user-images.githubusercontent.com/82287587/217076624-ecfe8947-816c-4ef3-8126-1e2f5f4c311a.png)

However, I had an older version which I was able to change usbportcfg and internalportcfg, so I can use my external drives as internal storage, but I am trying this now with no results. Can anyone help me please! 
Thank you so much!

**Comment by w84no1** [comment]
Yes, this would be awesome. Would love to be able to upgrade my DSM 6.2 to 7.1, but all of my disks are USB.

**Comment by 5olu7ion** [comment]
This already worked with 0.5 alpha. Today I upgraded to the newest 1.1 beta 2a and my settings are no longer working. synoinfo resets itsself, even though my user_config.json has the same setings as I had with 0.5 alpha. Relevant Keys are: internalportcfg, usbportcfg, esataportcfg and maxdisks from synoinfo section and sataportmap and diskidxmap from cmdline.
